controls/bsi_sys_1_6.yml

```diff
@@ -533,30 +533,40 @@ controls:
     levels:
     - elevated
     description: >-
-      The behaviour of containers and the applications or services running in them SHOULD be
-      monitored. Deviations from normal behaviour SHOULD be noticed and reported. The reports
+      (1) The behaviour of containers and the applications or services running in them SHOULD be
+      monitored. (2) Deviations from normal behaviour SHOULD be noticed and reported. (3) The reports
       SHOULD be handled appropriately in a centralised process for security incident handling.
-      At minimum, the behaviour to be monitored SHOULD cover:
-      • Network connections
-      • Created processes
-      • File system access attempts
-      • Kernel requests (syscalls)
+      (4) At minimum, the behaviour to be monitored SHOULD cover:
+        (5) • Network connections
+        (6) • Created processes
+        (7) • File system access attempts
+        (8) • Kernel requests (syscalls)
     notes: >-
-      ToDo
-    status: manual
-    #rules:
+      The requirement is not met by OpenShift itself but requires the implementation of additional 
+      container runtime security solutions such as Red Hat Advanced Cluster Security (ACS).
+
+      Note: At the host level, Red Hat CoreOS supports auditd, which is enabled by default. 
+      Policies for auditd can include network connections, created processes, file accesses and syscalls. 
+      Red Hat CoreOS provides many sample policies that cover all of the areas described.
+    status: does not meet
```

Comment:
I think it is not that appropriate. The requirement could be adressed by multiple solutions, among them ACS.

However, if a different solution such as Aqua Security is used, it could still be compliant.